packages/chain-adapters/src/evm/arbitrum/ArbitrumChainAdapter.test.ts

```diff
@@ -20,6 +21,98 @@ vi.mock('../../utils/validateAddress', () => ({
   validateAddress: vi.fn(),
 }))
 
+vi.mock('ethers', async importActual => {
```

Comment:
can we not update the custom module mock accordingly to avoid putting this everywhere?

Reply:
https://github.com/shapeshift/web/pull/6224/commits/043cbbfc7f388a359509c35aaeeaa1ad4f24c0c9